website/i18n/zh/docusaurus-plugin-content-blog/2022/03/03/develop-apisix-with-gitpod.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "开源云 IDE 产品新宠儿，如何使用 GitPod 开发 API 网关 Apache APISIX？"
```

Comment:
If it needs to be capitalized, gitpod will write it like this `Gitpod`. All such errors throughout the text should be revised.

![image](https://user-images.githubusercontent.com/8078418/156715412-e1c2f7e4-08ed-40d6-a9bd-64da96f1b03a.png)